tests/oss-fuzz/build.sh

```diff
@@ -19,23 +19,78 @@
 # https://github.com/google/oss-fuzz/blob/master/projects/libavif/build.sh
 # It builds the different fuzz targets.
 
+# To test changes to this file:
+# - make changes and commit to your REPO
+# - run:
+#     git clone --depth=1 git@github.com:google/oss-fuzz.git
+#     cd oss-fuzz
+# - modify projects/libavif/Dockerfile to point to your REPO
+# - run:
+#     python3 infra/helper.py build_image libavif
+#     # enter 'y' and wait for everything to be downloaded
+# - run:
+#     python3 infra/helper.py build_fuzzers --sanitizer address libavif
+#     # wait for the tests to be built
+# And then run the fuzzer locally, for example:
+#     python3 infra/helper.py run_fuzzer libavif \
+#     avif_fuzztest_enc_dec_incr@EncodeDecodeAvifFuzzTest.EncodeDecodeGridValid \
+#     --sanitizer address
+
+ln -s $SRC/fuzztest $SRC/libavif/ext/fuzztest
+
 # build dependencies
-cd ext && bash dav1d.cmd && bash libyuv.cmd && cd ..
+cd ext && bash aom.cmd && bash dav1d.cmd && bash googletest.cmd && bash libjpeg.cmd && \
+      bash libsharpyuv.cmd && bash libyuv.cmd && bash zlibpng.cmd && cd ..
+export CXXFLAGS="${CXXFLAGS} -DFUZZTEST_COMPATIBILITY_MODE"
 
 # build libavif
 mkdir build
 cd build
-cmake -G Ninja -DBUILD_SHARED_LIBS=OFF \
-    -DAVIF_CODEC_DAV1D=LOCAL -DAVIF_LIBYUV=LOCAL ..
+cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DAVIF_CODEC_AOM=LOCAL -DAVIF_CODEC_DAV1D=LOCAL \
+      -DAVIF_CODEC_AOM_DECODE=ON -DAVIF_CODEC_AOM_ENCODE=ON \
```

Comment:
Set `AVIF_CODEC_AOM_DECODE` to `OFF`.

Reply:
I am doing it but we want to test all stable decoding libraries no ? I know libaom has its own fuzzers but we could kill two birds with one stone by letting it ON no?